src/khoj/routers/helpers.py

```diff
@@ -553,7 +553,7 @@ async def generate_online_subqueries(
 
 async def schedule_query(
     q: str, conversation_history: dict, user: KhojUser, query_images: List[str] = None, tracer: dict = {}
-) -> Tuple[str, ...]:
+) -> Tuple[str, str, str]:
```

Comment:
Let's use `clean_json` instead of `json.loads` here for more graceful interpretation of the results.